docs/content/operator/common/genesis.mdx

```diff
@@ -1,19 +1,11 @@
----
-title: Genesis
-tags: [node-operation]
-teams:
-  - iotaledger/node
----
+
```

Comment:
Missing front matter again.

Reply:
The files in `common/` are shared-contents imported in files in `full-node/configs` and `validator-node/configs`, so front matter is added to `full-node/configs` and `validator-node/configs`.